### PR TITLE
Sinopia Search Results Improvments

### LINKED ISSUE #1412: Update Sinopia search result table
Including adding copy button and edit button.

See https://github.com/LD4P/sinopia/issues/255#issuecomment-537594034 for the table design (but ignore the rest of the page).

### PR DESCRIPTION
**Pull request by jermnelson**
Implements design of new Sinopia Search Results table as illustrated in #1412 
Screen Shot 2019-10-24 at 9 25 45 AM

Fixes #1412

### CODE REVIEW

File: src/components/search/SinopiaSearchResults.jsx [hunk 1]
```diff
@@ -54,15 +89,24 @@ const SinopiaSearchResults = (props) => {
         <div className="col-sm-2"></div>
         <div className="col-sm-8">
           <h3>Your List of Bibliographic Metadata Stored in Sinopia</h3>
```
[the remark this answers is not in the document]
Comment (jermnelson) [reply src/components/search/SinopiaSearchResults.jsx]: @michelleif how about **Listing of Bibliographic Metadata Stored in Sinopia**?

[the remark this answers is not in the document]
Comment (justinlittman) [reply src/components/search/SinopiaSearchResults.jsx]: Or nothing, like it is for QA search results.

File: src/components/search/SinopiaSearchResults.jsx [hunk 2]
```diff
@@ -29,15 +45,34 @@ const SinopiaSearchResults = (props) => {
...
 
   // Generates an HTML row
-
+  // TODO: Turn this function into a functional component
   const generateRows = () => {
     const rows = []
     props.searchResults.forEach((row, _index) => {
-      const rowIndex = _index + 1
       const link = `${Config.sinopiaServerBase}/${row.uri}`
       rows.push(<tr key={_index}>
```
[the remark this answers is not in the document]
Comment (jermnelson) [reply src/components/search/SinopiaSearchResults.jsx]: Done, using `shortid` instead.

File: src/components/search/SinopiaSearchResults.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jermnelson) [reply src/components/search/SinopiaSearchResults.jsx]: Removed this text.

File: src/components/search/SinopiaSearchResults.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinlittman) [reply src/components/search/SinopiaSearchResults.jsx]: Short id is bad also, because it is different every time. You should use the URI.

**Pull request merged**